rhc-ose-ansible/roles/dns-server/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Install bind and bind-utils
+  action: "{{ ansible_pkg_mgr }} name={{item}} state=latest"
+  with_items:
+  - bind
+  - bind-utils
+
+- name: Enable named
+  service: name=named enabled=yes
+
+- name: Install firewalld
+  action: "{{ ansible_pkg_mgr }} name=firewalld state=latest"
+
+- name: Enable firewalld
+  service: name=firewalld enabled=yes state=started
+
+- name: Open Firewall for DNS
+  firewalld: port={{item}} permanent=yes state=enabled immediate=yes
+  with_items:
+  - 53/tcp
+  - 53/udp
+
+- name: Setup Zone Directory
+  file: dest=/var/named/static state=directory owner=named group=named mode=0770
+
+- name: Setup ACLs
+  template: src=named.conf.acl.j2 dest=/etc/named/named.conf.acl owner=named group=named mode=0660
```

Comment:
to help readability these longer `template` lines could be reformatted to look like this:
```
    template: 
         src: named.conf.acl.j2
         dest: /etc/named/named.conf.acl 
         owner: named 
         group: named 
         mode: 0660 
```